Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/check_mark_button.tsx

```diff
@@ -7,24 +7,24 @@ interface Properties {
   size: number | string;
 
   /** The onClick event handler. */
-  onClick(event?: React.MouseEvent<any>): void;
+  onClick?(event?: React.MouseEvent<any>): void;
 
   /** Determines if the checkmark is currenly active(green) or inactive(grey) */
   isChecked: boolean;
 }
 
 /** Displays a check mark button. */
 export class CheckMarkButton extends React.Component<Properties, {}> {
-  static defaultProps = {
+
```

Comment:
Remove this blank line, the fact that this declaration is indented suffices to differentiate this line from the one above it. The request to put a blank line after the declaration is to clearly differentiate it from the declaration of the constructor. It seems picky but in the long run it helps make code more uniform/readable.

Reply:
The spacing is fine then right? I just need to remove the blank line on line 18?